web/src/app/pages/CalculationsPage.tsx

```diff
@@ -0,0 +1,75 @@
+import { useQuery } from '@apollo/react-hooks';
+import { gql } from 'apollo-boost';
+import React from 'react';
+
+import { CalculationsContainer } from '../components/content/calculations/CalculationsContainer';
+import { IStandardTableRow } from '../components/elements/Tables';
+import { H2, P } from '../components/elements/Texts';
+import { PageContent } from '../components/ui/PageContent';
+import { TitleBlock } from '../components/ui/TitleBlock';
+import { StPageWrapper } from './CalculatorPage';
+
+export interface ICalculationInfo {
+  id: string,
+  ship: {
+    id: string,
+    name: string,
+    country: string,
+  }
+}
+
+interface ICalculationsData {
+  calculations: ICalculationInfo[],
+}
+
+export const CalculationsPage: React.FunctionComponent = (props: any) => {
```

Comment:
Do we need a type for props? Shouldn't that be inferred from the type of the function/component?